localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -280,6 +280,22 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if(state_ptr_->count("nearest_voxel_transformation_likelihood") &&
+      std::stoi((*state_ptr_)["nearest_voxel_transformation_likelihood"]) < 2.3) {
```

Comment:
Could you please consider implementing 2.3 as a const. 
As Koji said, it is better to make these ROS 2 parameters.

Reply:
Sure, but i have a question. All the other if statements' conditions implemented as literal values. Do you want me to convert them to ROS 2 parameters?